TODO-rebase.sh

```diff
@@ -206,26 +206,26 @@ run_specific_rebase () {
 		export GIT_EDITOR
 		autosquash=
 	fi
-	. git-rebase--$type
+	# . git-rebase--$type
 
```

Comment:
How about commenting out this line, too? That would make it slightly easier to read on GitHub, I think.